loc/de_de.json

```
@@ -295,37 +206,44 @@
     "input_done": "Fertig",
     "input_paste": "Einfügen",
     "input_total": "Gesamt:",
-    "no_tx_signing_in_progress": "Keine Transaktionsignierung in Arbeit",
-    "open_settings": "Einstellungen Öffnen",
     "permission_camera_message": "Wir brauchen Deine Erlaubnis um die Kamera zu nutzen.",
     "permission_camera_title": "Erlaubnis die Kamera zu benutzen",
-    "permission_storage_denied_message": "BlueWallet kann die Datei nicht speichern. Bitte öffne die Systemeinstellungen und erteile der App BlueWallt das Recht den internen Speicher zu verwenden.",
+    "open_settings": "Einstellungen Öffnen",
     "permission_storage_later": "Später beantworten",
     "permission_storage_message": "BlueWallet braucht zur Speicherung dieser Datei die Erlaubnis auf den internen Speicher zuzugreifen.",
+    "permission_storage_denied_message": "BlueWallet kann die Datei nicht speichern. Bitte öffne die Systemeinstellungen und erteile der App BlueWallt das Recht den internen Speicher zu verwenden.",
     "permission_storage_title": "Speicherzugriffsrecht",
-    "problem_with_psbt": "PSBT Problem",
     "psbt_clipboard": "In die Zwischenablage kopieren",
     "psbt_this_is_psbt": "Dies ist eine partiell signierte Bitcoin-Transaktion (PSBT). Bitte signiere sie mithilfe Deiner Hardware-Wallet.",
     "psbt_tx_export": "In Datei exportieren",
+    "no_tx_signing_in_progress": "Keine Transaktionsignierung in Arbeit",
     "psbt_tx_open": "Öffne signierte Transaktion.",
     "psbt_tx_scan": "Signierte Transaktion scannen",
     "qr_error_no_qrcode": "Das gewählte Foto enthält keinen QR code",
     "qr_error_no_wallet": "Die ausgewählte Datei enthält keine Wallet, die importiert werden kann.",
     "success_done": "Fertig",
-    "txSaved": "Die Transaktionsdatei ({filePath}) wurde im Download-Ordner gespeichert."
+    "txSaved": "Die Transaktionsdatei ({filePath}) wurde im Download-Ordner gespeichert.",
+    "problem_with_psbt": "PSBT Problem"
```

```suggestion
    "problem_with_psbt": "PSBT-Problem"
```

This is something that many people do wrong, in German nouns are always combined (either with no space in between and only the first letter of the resulting word uppercase or connected with a hyphen)